scripts/docker-compose.showcase.yml

```diff
@@ -7,6 +7,7 @@ services:
     depends_on:
```

Comment:
Wondering if it makes sense to combine everything into one docker-compose.yml?

Reply:
Let's discuss w/team. I think they should be separate because it's easier to fire up the "core" if I was using this to develop my own LOB against. We can achieve a lot with profiles... so whatever the rest of the team wants, I'm ok with. 6 of 1...